sdk/timeseriesinsights/Instance_API_Design.md

```diff
@@ -0,0 +1,64 @@
+# Time Series Insights
+
+## Instances
+Time Series Model instances are virtual representations of the time series themselves. Instances have descriptive information associated with them called instance properties, such as a time series ID, type, name, description, hierarchies, and instance fields. At a minimum, instance properties include hierarchy information.
+
+### GET /timeseries/instances
+
+```csharp
+public virtual Pageable<TimeSeriesInstance> GetInstances(CancellationToken cancellationToken = default);
+```
+
+### POST /timeseries/instances/suggest
+
+```csharp
+public virtual Response<InstancesSearchStringSuggestion[]> GetSearchSuggestions(InstancesSuggestRequest instancesSuggestRequest, CancellationToken cancellationToken = default);
+
+// Thoughts on renaming InstancesSuggestRequest to GetSearchSuggestionsOptions
+// Thoughts on renaming InstancesSearchStringSuggestion to InstanceSuggestion
+```
+
+### POST /timeseries/instances/$batch
+
+```csharp
+public virtual Response<InstanceOrError[]> GetInstances(TimeSeriesId[] timeSeriesIds, CancellationToken cancellationToken = default);
```

Comment:
can we not use list here for timeSeriedIds?

Reply:
I will use IEnumerable<TimeSeriesId> per David's suggestion during the design meeting.